### PR TITLE
feature(website): overview section on new landing page

### PR DESCRIPTION
**Pull request by DarkMenacer**

### CODE REVIEW

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx [hunk 1]
```diff
@@ -10,11 +10,11 @@ export async function Overview({ lang, region }: DefaultParams) {
...
-		<BaseContainer className="mb-48 flex flex-col items-center justify-center">
-			<Typography size="3xl" weight="medium" color="foreground-dark" className="my-10 w-3/5 text-center">
+		<BaseContainer className="theme-v2 mb-16 flex flex-col items-center justify-center md:mb-48">
+			<Typography size="3xl" weight="medium" className="my-10 w-full text-center md:w-3/5">
 				{translator.t('section-2.title-1')}
 			</Typography>
-			<div className="mb-8 w-3/5 text-center">
+			<div className="mb-8 w-full text-center md:w-3/5">
 				{translator.t<{ text: string; color?: FontColor }[]>('section-2.title-2').map((title, index) => (
 					<Typography
```
Comment (mkue) [inline website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx:19]: the `className` prop is probably not needed here

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/monthly-income.tsx [hunk 2]
```diff
@@ -10,7 +10,10 @@ export async function MonthlyIncome({ lang, region }: DefaultParams) {
 	});
 
 	return (
-		<BaseContainer backgroundColor="bg-foreground-dark" className="py-3">
+		// <BaseContainer backgroundColor="bg-foreground-dark" className="py-3">
+		<BaseContainer className="theme-blue-v2 py-3">
+			{' '}
+			{/* Is there a way to use theme-blue-v2 to set background blue? */}
```
Comment (mkue) [inline website/src/app/[lang]/[region]/v2/(home)/(sections)/monthly-income.tsx:16]: This comment is not needed anymore, right? You are setting the theme now...

**Review by mkue — APPROVED** [review]
nice! looks good :-)

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx (hunk 1 above)
Comment (DarkMenacer) [reply website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx:19]: Won't we need it for responsiveness?

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/monthly-income.tsx (hunk 2 above)
Comment (DarkMenacer) [reply website/src/app/[lang]/[region]/v2/(home)/(sections)/monthly-income.tsx:16]: This wasn't working as expected on my local system or on the deployed one either, is it on yours?

Screenshot 2024-07-21 at 20 09 40

Ideally it should be screen-wide but that is not happening for some reason

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx (hunk 1 above)
Comment (mkue) [reply website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx:19]: You don't need it because you already set it on the parent element.

**Pull request merged**